types/three/three-core.d.ts

```diff
@@ -6643,10 +6643,9 @@ export namespace SceneUtils {
 }
 
 export namespace ShapeUtils {
-    export function area(contour: number[]): number;
-    export function triangulate(contour: number[], indices: boolean): number[];
-    export function triangulateShape(contour: number[], holes: any[]): number[];
-    export function isClockWise(pts: number[]): boolean;
+    export function area(contour: Vector2[]): number;   
+    export function triangulateShape(contour: Vector2[], holes: any[]): number[];
+    export function isClockWise(pts: Vector2[]): boolean;
```

Comment:
The spelling is `isClockwise`

https://threejs.org/docs/#api/extras/ShapeUtils.isClockwise

Reply:
Hi,
it seems THREE documentation is wrong.
Please look at the sources:
https://github.com/mrdoob/three.js/blob/master/src/extras/ShapeUtils.js line 26

isClockWise: function ( pts ) {